Source/engine/displacement.hpp

```diff
@@ -11,143 +11,104 @@
 
 namespace devilution {
 
-struct Displacement {
-	int deltaX;
-	int deltaY;
+template <typename DeltaT>
+struct DisplacementOf;

```

Comment:
Maybe `DisplacementOff` would be better? off is more common shortening of offset, and DisplacementOf reads as 'displacement of something'

Reply:
I assume it's supposed to be `DisplacementOf<int>` = "displacement of int" in a similar fashion to how you might read `std::vector<int>` as "vector of int". The word "of" seems to be a bit redundant in this case, but that's probably because it would otherwise conflict with the alias established by `using Displacement = DisplacementOf<int>;`

Reply:
Yep, that's exactly it